masterlist.yaml

```diff
@@ -1843,21 +1853,130 @@ plugins:
         name: 'Immersive Citizens - AI Overhaul SE on Nexus Mods'
       - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/3016198'
         name: 'Immersive Citizens - AI Overhaul [PC] on Bethesda.net'
-    group: *alternateStartGroup
+    inc:
+      - name: 'iWil_BelethorsGoodStore.esp'
+        display: 'Belethors Overrated Goods'
+      - 'BetterCitiesWhiterun.esp'
+      - name: 'Blues Skyrim.esp'
+        display: '[Dawn of Skyrim (Director''s Cut) SE](https://www.nexusmods.com/skyrimspecialedition/mods/9074)'
+      - 'BosmericDrunkenHuntsmanReborn.esp'
+      - 'EEKs Arcadia''s Cauldron.esp'
+      - 'EEKs Arcadia''s - OCS.esp'
+      - name: 'Holds.esp'
+        display: '[Holds The City Overhaul](https://www.nexusmods.com/skyrimspecialedition/mods/10609)'
+      - 'InnCredible.esp'
+      - name: 'iWil_UselessShopOverhaul.esp'
+        display: 'The Useless Shop and Interior Overhaul'
+      - 'NerniesCityandVillageExpansion.esp'
+      - 'Northern Bathhouses.esp'
+      - name: 'Open Cities Skyrim.esp'
+        condition: 'active("Open Cities Skyrim.esp") and not active("Immersive Citizens - OCS patch.esp")'
+        display: '[Open Cities Skyrim](https://www.nexusmods.com/skyrimspecialedition/mods/281)'
+      - 'RealWarMaiden.esp'
+      - 'ScottishBanneredMare.esp'
+      - 'Skyrim Radioactive -.*\.esp'
+      - name: 'tpos_ultimate_esm.esp'
+        display: '[The People Of Skyrim Complete SSE](https://www.nexusmods.com/skyrimspecialedition/mods/13196)'
     after:
       - '3DNPC.esp'
-      - 'Alternate Start - Live Another Life.esp'
```

Comment:
I checked for conflicts and there are only a handful, and even then, those overwrites are minor.